.gitignore

```diff
@@ -1 +1,32 @@
-.vscode/
\ No newline at end of file
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
```

Comment:
Do we have DLL's we are compiling?

Reply:
Not sure if you are commenting on the dll comment like the previous comment or the pch.  I have absolutely no clue if we will use .pch -  pch stands for precompiled header. In computer programming, a precompiled header is a (C or C++) header file that is compiled into an intermediate form that is faster to process for the compiler. Usage of precompiled headers may significantly reduce compilation time, especially when applied to large header files...

Again, you are welcome to remove all of the .gitignore content if you want.  I certainly won't lose sleep over it.  Like I think I mentioned, I haven't explored github code before now and I thought it was cool that someone put a sample .gitignore together specifically for c++.

Reply:
So we only need to put in gitignores for anything we are building that we do not need to commit. If we started building DLL's for our application we would include it. For now we do not need to include any extensions we are not using when we build. We just need to keep it simple.

Reply:
I don't find anything in the .gitignore complicated.

Reply:
Good, then just put the basics of  the files  we need to ignore